.github/workflows/eamxx-v1-testing.yml

```diff
@@ -71,6 +71,8 @@ jobs:
             short_name: ERS_Ln22.ne4pg2_ne4pg2.F2010-SCREAMv1.scream-small_kernels--scream-output-preset-5
           - full_name: SMS_D_Ln5.ne4pg2_oQU480.F2010-SCREAMv1-MPASSI.ghci-snl-cpu_gnu.scream-mam4xx-all_mam4xx_procs
             short_name: SMS_D_Ln5.ne4pg2_oQU480.F2010-SCREAMv1-MPASSI.scream-mam4xx-all_mam4xx_procs
+          - full_name: SMS_Ln3_P4.ne4pg2_oQU480.F2010-MMF2
```

Comment:
Good question, I am not entierly sure. I don't recall if I set the regex to recognize the ghci-snl-cpu machine in the machine config file. Maybe, maybe not. I'll add the machine just in case.